klippy/extras/resonance_tester.py

```diff
@@ -178,7 +178,10 @@ def cmd_TEST_RESONANCES(self, gcmd):
                             "%s-axis accelerometer measured no data" % (
                                 chip_axis,))
                 new_data = helper.process_accelerometer_data(chip_values)
-                data = data.join(new_data) if data else new_data
+                if data is None:
+                    data = new_data
+                else:
+                    data.join(new_data)
```

Comment:
Should this be `data = data.join(new_data)`?

-Kevin

Reply:
No, this is exactly the bug I am trying to fix. join method joins in place and does not return anything.

Reply:
Ah, okay.  The standard python string has a `join()` method which returns the new value.  Perhaps we should rename `CalibrationData.join()` to `CalibrationData.extend()` so that it better matches the Python method naming scheme?

-Kevin

Reply:
I renamed the method to `add_data`.